packages/devtools_app/lib/src/screens/network/har_network_data.dart

```diff
@@ -67,31 +75,28 @@ class HarDataEntry {
   factory HarDataEntry.fromJson(Map<String, Object?> json) {
     _convertHeaders(json);
 
-    final modifiedRequestData =
-        _remapCustomFieldKeys(json) as Map<String, dynamic>;
+    final modifiedRequestData = _remapCustomFieldKeys(json);
 
     // Retrieving url, method from requestData
-    modifiedRequestData['uri'] =
-        (modifiedRequestData['request'] as Map<String, Object?>)['url'];
-    modifiedRequestData['method'] =
-        (modifiedRequestData['request'] as Map<String, Object?>)['method'];
+    final requestData = modifiedRequestData[NetworkEventKeys.request.name]
+        as Map<String, Object?>;
+    modifiedRequestData[NetworkEventKeys.uri.name] =
+        requestData[NetworkEventKeys.url.name];
+    modifiedRequestData[NetworkEventKeys.method.name] =
+        requestData[NetworkEventKeys.method.name];
 
     // Adding missing keys which are mandatory for parsing
-    (modifiedRequestData['response'] as Map<String, Object?>)['redirects'] = [];
+    final responseData = modifiedRequestData[NetworkEventKeys.response.name]
+        as Map<String, Object?>;
+    responseData[NetworkEventKeys.redirects.name] = <Map<String, Object?>>[];
     Object? requestPostData;
     Object? responseContent;
-    if (modifiedRequestData['response'] != null &&
-        (modifiedRequestData['response'] as Map<String, Object?>)['content'] !=
-            null) {
-      responseContent =
-          (modifiedRequestData['response'] as Map<String, Object?>)['content'];
+    if (responseData[NetworkEventKeys.content.name] != null) {
+      responseContent = responseData[NetworkEventKeys.content.name];
     }
 
-    if (modifiedRequestData['request'] != null &&
-        (modifiedRequestData['request'] as Map<String, Object?>)['postData'] !=
-            null) {
-      requestPostData =
-          (modifiedRequestData['response'] as Map<String, Object?>)['content'];
+    if (requestData[NetworkEventKeys.postData.name] != null) {
+      requestPostData = responseData[NetworkEventKeys.content.name];
```

Comment:
I think the wrong key is being used here. This is for postData, not content.

Reply:
good catch, I think during re-factoring i made this mistake, I have fixed it now.